Sources/Core/ISO8601.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+// Modified from http://stackoverflow.com/a/28016692/239380
+
+public struct ISO8601 {
+    public static let shared = ISO8601()
+    public let formatter: DateFormatter
+
+    public init() {
+        formatter = DateFormatter()
+        formatter.calendar = Calendar(identifier: .iso8601)
+        formatter.locale = Locale(identifier: "en_US_POSIX")
```

Comment:
Hmm, while looking I found this: http://stackoverflow.com/a/32459234/239380, which links to this technical note from Apple: https://developer.apple.com/library/content/qa/qa1480/_index.html

> In most cases the best locale to choose is "en_US_POSIX", a locale that's specifically designed to yield US English results regardless of both user and system preferences. "en_US_POSIX" is also invariant in time (if the US, at some point in the future, changes the way it formats dates, "en_US" will change to reflect the new behaviour, but "en_US_POSIX" will not), and between machines ("en_US_POSIX" works the same on iOS as it does on OS X, and as it it does on other platforms).